### PR TITLE
Add University

### PR DESCRIPTION
**Pull request by Victorybrown**
- Duke University
- Rochester Institute of Technology
- University of Vermont
- University of Michigan

### CODE REVIEW

File: universities/university-of-missouri.md [hunk 1]
```diff
@@ -1 +1 @@
-# University of Missouri
```
Comment (RichardLitt) [inline universities/university-of-missouri.md:1]: What's going on here?

File: universities/university-of-vermont.md [hunk 2]
```diff
@@ -1 +1,31 @@
+---
+title: "University of Vermont"
```
Comment (RichardLitt) [inline universities/university-of-vermont.md:2]: We already have this in a PR. Can you check that version against this and see what you could've done differently?

File: universities/university-of-michigan.md [hunk 3]
```diff
@@ -1 +1,25 @@
...
+- *Link*: https://open.umich.edu/
+- *Member of*: [HELIOS](https://www.heliosopen.org/members)
+
+## General Description
+
+> Open Michigan is an open arm of the University of Michigan community that makes the products of the university; its research, teaching, and creative work available to the world beyond campus.
+
+## Core Objectives
+
+-
```
Comment (RichardLitt) [inline universities/university-of-michigan.md:19]: What's going on here?

**Review by RichardLitt — CHANGES_REQUESTED** [review]
A good start! Some comments.

**Comment by RichardLitt** [comment]
@Victorybrown I think one PR per file will work easier. Can you do that? :)

**Comment by Victorybrown** [comment]
Okay, will do that

**Comment by RichardLitt** [comment]
Let's split this out into new PRs. Can you do that? 

Here are some steps to help: 

1. `git checkout main` - go to the main branch.
2. And then `git pull origin main` - pull changes. 
2. `git checkout -b new-branch` - Make a new branch. Replace `new-branch` with the name of the university.
3. `git checkout academic-map-two -- universities/university-of-colorado.md` - this should checkout the file from this PR. Then, edit the file and save it. 
4. `git push origin new-branch` - push it up and make a new PR.

Make sense?

**Pull request closed without merging**